### DISCUSSION TITLE
V8 Installation

### DISCUSSION THREAD

**Discussion by redmorello** [Infocaster/UrlTracker#42] [Q&A] [answered]
When installing this package on V8, should I be turning off the built in Umbraco Redirect Url Management functionality? (via the Disable URL tracker button)

**Comment by D-Inventor** [comment, marked as answer]
Hi @redmorello ! If you rely on the Umbraco Redirect Url Management functionality, it shouldn't be necessary to disable it. At least I don't think it conflicts with the built in url management tool. It is noteworthy though that the URL Tracker offers every feature that the built-in url management tool has and more, so disabling it is highly recommended!
